src/components/hero.js

```diff
@@ -75,6 +76,12 @@ const Arrow = ({ color, size }) => {
 const Banner = () => <ImgContainer data={banner}>Banner</ImgContainer>
 
 const Hero = ({ title, background }) => {
+  useEffect(() => {
+    if (process.env.NODE_ENV !== "development") {
+      amplitudeInit()
```

Comment:
Why do this in Hero? Can this be done in index.js instead?

Reply:
Yeah, definitely. Moving it now